auto_tests/BUILD.bazel

```diff
@@ -8,8 +8,13 @@ cc_library(
 
 cc_library(
     name = "run_auto_test",
```

Comment:
Can you rename this to auto_test_support? Right now, all the "_test.c" files are actually tests. We use that fact to generate build/test rules for them. Either this, or add run_auto_test to an exclusion list below where we create those rules.